docs/technical_reference/property_models/activated_sludge_model.rst

```diff
@@ -0,0 +1,144 @@
+ASM1 Property Package
+=====================
+
+This package implements property relationships for the treatment of activated sludge as provided in `Alex, J. et al. (2008) <https://www.iea.lth.se/publications/Reports/LTH-IEA-7229.pdf>`_.
+
+This Activated Sludege Model no.1 (ASM1) property package:
+   * supports only 'H2O', 'TDS', 'TSS', and 'Sludge' as Components
+   * supports only liquid phase
+   * is formulated on a mass basis
+   * does NOT support formulations on a molar basis
+   * includes mass density correction for fraction of suspended/dissolved solids
+
+Sets
+----
+.. csv-table::
+  :header: "Description", "Symbol", "Indices"
+
+  "Components", ":math:`j`", "['H2O', 'TDS', 'TSS', 'Sludge']"
+  "Phases", ":math:`p`", "['Liq']"
+
+State variables
+---------------
+.. csv-table::
+   :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+   "Component mass flowrate", ":math:`M_j`", "flow_mass_phase_comp", "[p, j]", ":math:`\text{kg/s}`"
+   "Temperature", ":math:`T`", "temperature", "None", ":math:`\text{K}`"
+   "Pressure", ":math:`P`", "pressure", "None", ":math:`\text{Pa}`"
+
+Parameters
+----------
+.. csv-table::
+ :header: "Description", "Symbol", "Parameter", "Index", "Units"
+
+ "Fluid specific heat capacity", ":math:`c_p`", "cp", "None", ":math:`\text{J/kg/K}`"
+ "Reference density (at 273 K)", ":math:`\rho_{ref}`", "ref_dens_liq", "None", ":math:`\text{kg/}\text{m}^3`"
+ "Relative density contribution from salt mass fraction", ":math:`\rho_s`", "dens_slope", "None", ":math:`\text{kg/}\text{m}^3`"
+ "First density temperature coefficient", ":math:`A`", "dens_param_A", "None", ":math:`\text{K}^{-2}`"
+ "Second density temperature coefficient", ":math:`B`", "dens_param_B", "None", ":math:`\text{K}^{-1}`"
+ "Third density temperature coefficient", ":math:`C`", "dens_param_C", "None", ":math:`\text{dimensionless}`"
+ "First pressure coefficient", ":math:`\alpha`", "ref_pressure_correction", "None", ":math:`\text{dimensionless}`"
+ "Second pressure coefficient", ":math:`\beta`", "ref_pressure_slope", "None", ":math:`\text{Pa}^{-1}`"
+ "Reference viscosity (at 273 K)", ":math:`\mu_{ref}`", "mu_A", "None", ":math:`\text{kg/}\text{m/}\text{s}`"
+ "First viscosity temperature coefficient", ":math:`\mu_B`", "mu_B", "None", ":math:`\text{K}`"
+ "Second viscosity temperature coefficient", ":math:`\mu_C`", "mu_C", "None", ":math:`\text{K}`"
```

Comment:
You can remove anything here that is not in the ASM1 property model. Be sure to add anything from the ASM1 prop model that is missing here.